crawler/pom.xml

```diff
@@ -27,6 +27,7 @@ the License.
 	<artifactId>cas-crawler</artifactId>
 	<name>Catalog and Archive Crawling Framework</name>
 	<description>The Catalog and Archive Service Crawling Framework.</description>
+	<packaging>bundle</packaging>
```

Comment:
Makes me sad I spent hours copying and pasting!

But your plan also doesn't fit my idea because bundling all of oodt into a
mega bundle is a bloat that isn't required. For example I just want to
deploy a file manager and expose it via DOSGI to a main server, why would I
deploy a massive jar for a very light weight requirement.

I started by creating a wrapper jar for it but I figured as we're only
changing a very default manifest file actually pushing it down to the Base
jars made more sense.
On 23 Jul 2015 18:48, "Chris Mattmann" <notifications@github.com> wrote:

> In crawler/pom.xml
> <https://github.com/apache/oodt/pull/23#discussion_r35350102>:
>
> > @@ -27,6 +27,7 @@ the License.
> >  	<artifactId>cas-crawler</artifactId>
> >  	<name>Catalog and Archive Crawling Framework</name>
> >  	<description>The Catalog and Archive Service Crawling Framework.</description>
> > +	<packaging>bundle</packaging>
>
> maybe we shouldn't change the default packaging per project, but instead
> create an oodt-bundle that is packaged as an OSGI bundle that combines the
> constituent sub projects.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/oodt/pull/23/files#r35350102>.
>